### PR TITLE
[ADD]: New module l10n_ec_ote

### PR DESCRIPTION
**Pull request by gomezgleonardob**
**Draft Module L10n_ec_Ote**

- Inherit Res.City 
-  Add csv Parish , City
- Add Field Parish in Model (Partner, Company)

### CODE REVIEW

File: l10n_ec_ote/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    'name': 'OTE for Ecuador',
+    'depends': ['base', 'sales_team','base_address_city'],
+    'category': 'Localization',
+    'version': '15.0.0.1.0',
```
Comment (celm1990) [inline l10n_ec_ote/__manifest__.py]: Initial version must be 15.0.1.0.0 according [OCA guideslines ](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#12version-numbers)
```suggestion
    'version': '15.0.1.0.0',
```

File: l10n_ec_ote/data/res.bank.csv [hunk 2]
```diff
@@ -0,0 +1,93 @@
+id,country:id,name,bic,active
```
Comment (celm1990) [inline l10n_ec_ote/data/res.bank.csv]: You can update your file and only keep records that don't currently exist in the odoo base, see file https://github.com/odoo/odoo/blob/15.0/addons/l10n_ec/data/res.bank.csv

File: l10n_ec_ote/data/res.country.state.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+id,country_id:id,code,name
+base.state_ec_90,base.ec,90,ZONA NO DELIMITADA
```
Comment (celm1990) [inline l10n_ec_ote/data/res.country.state.csv:2]: Because this record? this is not State than exist

File: l10n_ec_ote/models/res_company.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+from odoo import api, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    l10n_ec_company_parish_id = fields.Many2one(
+        "l10n.ec.parish",
+        ondelete="restrict",
+        string="Parish",
```
Comment (celm1990) [inline l10n_ec_ote/models/res_company.py:10]: Can be related of `partner_id`?

File: l10n_ec_ote/security/ir.model.access.csv [hunk 5]
```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_l10n_ec_parish,l10n.ec.parish,model_l10n_ec_parish,sales_team.group_sale_salesman,1,0,0,0
```
Comment (celm1990) [inline l10n_ec_ote/security/ir.model.access.csv]: Please add  group  `base.group_partner_manager` to `CRUD`
image

File: l10n_ec_ote/views/l10n_ec_parish_view.xml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+                <field name="code" />
+            </search>
+        </field>
+    </record>
+
+    <record id="l10n_ec_parish_form" model="ir.ui.view">
+        <field name="name">l10n.ec.parish.form</field>
+        <field name="model">l10n.ec.parish</field>
+        <field name="arch" type="xml">
+            <form string="Parroquia">
```
Comment (celm1990) [inline l10n_ec_ote/views/l10n_ec_parish_view.xml]: Remove string
```suggestion
            <form>
```

File: l10n_ec_ote/views/l10n_ec_parish_view.xml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+                </group>
+            </form>
+        </field>
+    </record>
+
+    <record id="l10n_ec_parish_tree" model="ir.ui.view">
+        <field name="name">l10n.ec.parish.tree</field>
+        <field name="model">l10n.ec.parish</field>
+        <field name="arch" type="xml">
+            <tree string="Parroquias">
```
Comment (celm1990) [inline l10n_ec_ote/views/l10n_ec_parish_view.xml]: same here
```suggestion
            <tree>
```

File: l10n_ec_ote/views/res_city_view.xml [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+                <field name="zipcode" />
+            </search>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="res_city_form">
+        <field name="name">res.city.form</field>
+        <field name="model">res.city</field>
+        <field name="arch" type="xml">
+            <form string="Cantón">
```
Comment (celm1990) [inline l10n_ec_ote/views/res_city_view.xml]: same here

File: l10n_ec_ote/views/res_city_view.xml [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+                </group>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="res_city_tree">
+        <field name="name">res.city.tree</field>
+        <field name="model">res.city</field>
+        <field name="arch" type="xml">
+            <tree string="Cantones">
```
Comment (celm1990) [inline l10n_ec_ote/views/res_city_view.xml]: and here

File: l10n_ec_ote/models/res_city.py [hunk 10]
```diff
@@ -0,0 +1,6 @@
+from odoo import models, fields , api
+
+class ResCity(models.Model):
+    _inherit='res.city'
+
+l10n_ec_res_city_id = fields.One2many('res.city',inverse_name='city_id',string="Cantón")
```
Comment (celm1990) [inline l10n_ec_ote/models/res_city.py]: Fix indentation

File: l10n_ec_ote/.gitignore [hunk 11]
```diff
@@ -0,0 +1 @@
+*.pyc
```
Comment (celm1990) [inline l10n_ec_ote/.gitignore]: this file is not needed here, please update if necesary into root folder https://github.com/Odoo-EC/l10n-ecuador/blob/15.0/.gitignore

**Review by celm1990 — CHANGES_REQUESTED** [review]
Thanks for contribution, first technical review and few changes

File: l10n_ec_ote/data/res.country.state.csv (hunk 3 above)
Comment (gomezgleonardob) [reply l10n_ec_ote/data/res.country.state.csv:2]: Some territories are not defined as belonging to a state or city 
https://es.wikipedia.org/wiki/Anexo:Cantones_de_Ecuador#Zonas_no_delimitadas

**Review by celm1990 — CHANGES_REQUESTED** [review]
@gomezgleonardob 
Please review you commits, module l10n_ec_account_edi is delete, becasuse???

**Pull request merged**